src/content-script/openai.js

```diff
@@ -55,9 +54,56 @@ export async function callGPT4(messages) {
     if (data.error) {
       throw new Error(data.error.message);
     }
-    return data.choices?.[0]?.message?.content;
+    const gptMessage = JSON.parse(data.choices?.[0]?.message?.content);
+    return { keyword: gptMessage.keyword, content: gptMessage.content };
   } catch (error) {
     console.error("OpenAI API Error:", error);
     throw error;
   }
 }
+
+export async function preprocessText(rawText) {
+  try {
+    const prompt = `
+      You receive some raw conversation information between an AI assistant and a candidate. You should summarize the information coming from the candidate in a way that creates stand-alone knowledge bits that can be stored for later consumption.
```

Comment:
We can move all prompts to dedicated folder and files. Since we are going to get more prompts. 
Not a blocker for this PR

Reply:
Also entire request function can be moved to separate util. Just to provide prompt and content.
Not a blocker. Just to polish code